mesh/confirmation/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+from . import views as confirmation_views
+
+urlpatterns = [
+    path('<str:user_email>/', confirmation_views.email_confirmation, name="email_confirmation"),
+    path('<str:user_email>/<str:token>/', confirmation_views.confirm_token, name="confirm_token"),
```

Comment:
Should we have a path specifically for emails? Like /emails/<str...

Reply:
I figure that the confirmation path in the urls.py in the root mesh folder where it is already confirmation/<str:user_email>/ handles this idea? Unless you mean add the /email/ on top of the confirmation path for clarity.